Core/include/Acts/Fitter/KalmanFitter.hpp

```diff
@@ -637,13 +637,16 @@ class KalmanFitter {
           return Result<void>::success();
         }
 
-        // Update state and stepper with pre material effects
-        materialInteractor(surface, state, stepper, preUpdate);
-
         // Transport & bind the state to the current surface
         auto [boundParams, jacobian, pathLength] =
             stepper.boundState(state.stepping, *surface);
 
+        // Update state and stepper with pre material effects
+        materialInteractor(surface, state, stepper, preUpdate);
+
+        // Update state and stepper with pre material effects
```

Comment:
I guess the duplication of `materialInteractor` is some typo?

Reply:
Right, copy&paste mistake